easybuild/easyconfigs/g/gomkl/gomkl-2021b.eb

```
@@ -0,0 +1,19 @@
+easyblock = "Toolchain"
+
+name = 'gomkl'
+version = '2021b'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain with OpenMPI and MKL"""
+
+toolchain = SYSTEM
+
+local_comp = ('GCC', '10.3.0')
```

The 2021b toolchain generation is using `GCC/11.2.0`:
```suggestion
local_comp = ('GCC', '11.2.0')
```

---

Fixed. Thanks for finding this, that was a really silly mistake I made.